plugin/src/main/java/io/jenkins/plugins/casc/RootElementConfigurator.java

```diff
@@ -33,6 +35,14 @@
                 configurators.add(new DescriptorConfigurator(descriptor));
         }
 
+        configurators.sort(Comparator.comparingDouble(c -> {
+            Extension extension = c.getClass().getAnnotation(Extension.class);
+            if (extension == null) {
+                return Double.MIN_VALUE;
```

Comment:
I don't understand the context, but this is a common pitfall, so I'll say it just in case:

Do you really want to use `Double.MIN_VALUE`? This is defined as the smallest possible **positive** double value that is closest to zero. Maybe you want to use something like `-Double.MAX_VALUE`?

Reply:
it could also be sorted as zero tbh.